app/models/concerns/slug_history.rb

```diff
@@ -24,7 +24,9 @@ def slug_history_key(slug)
     # Check that slug_attribute (defined by Slugged) was changed
     def save_slug_if_changed
       if previous_changes.keys.include? slug_source.to_s
-        self.set_slug!
+        if self.title?
```

Comment:
I think we'll need to remove this per the other PR (https://github.com/jellypbc/poster/pull/96)

Reply:
Resolved this as a merge conflict fix.